### PR TITLE
fix: deactivate the csvts NCNN test when caffe is not built

### PR DESCRIPTION
**Pull request by beniz**

### CODE REVIEW

File: tests/ut-ncnnapi.cc [hunk 1]
```diff
@@ -165,7 +166,7 @@ TEST(ncnnapi, service_lstm)
   mkdir(csvts_repo.c_str(), 0777);
   std::string sname = "my_service_csvts";
   std::string jstr
-      = "{\"mllib\":\"caffe\",\"description\":\"my ts "
+      = "{\"mllib\":\"ncnn\",\"description\":\"my ts "
```
Comment (fantes) [inline tests/ut-ncnnapi.cc:169]: the idea here was to learn and predict with caffe, then to compare to ncnn prediction, here you try to learn with ncnn ?

File: tests/ut-ncnnapi.cc [hunk 2]
```diff
@@ -216,7 +217,7 @@ TEST(ncnnapi, service_lstm)
   std::string jpredictstr
       = "{\"service\":\"" + sname
         + "\",\"parameters\":{\"input\":{\"timesteps\":999,\"connector\":"
-          "\"csvts\",\"scale\":true,\"min_vals\":"
+          "\"caffe\",\"scale\":true,\"min_vals\":"
```
Comment (fantes) [inline tests/ut-ncnnapi.cc:220]: this makes me core dump : "connector":"caffe" ?

**Review by fantes — CHANGES_REQUESTED** [review]

**Comment by beniz** [comment]
Thank you, these are typos, I went too fast.

**Review by fantes — APPROVED** [review]

**Review by BynaryCobweb — APPROVED** [review]

**Comment by sileht** [comment]
@mergifyio update

**Pull request merged**